xml/admin_nfsganesha.xml

```diff
@@ -389,6 +405,80 @@ EXPORT {
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="ganesha-getting-exports">
+   <title>Getting Exports Configuration</title>
+   <para>
+    To get the exports configuration details, follow these steps:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Get the &rados; pool name and namespace for &ganesha; exports. The
+      following command outputs a string of the
+      <replaceable>POOL_NAME</replaceable>/<replaceable>NAMESPACE</replaceable>
+      form.
+     </para>
+<screen>&prompt.cephuser;ceph dashboard get-ganesha-clusters-rados-pool-namespace
+cephfs_data/ganesha</screen>
+    </step>
+    <step>
+     <para>
+      Given the pool name and namespace, list the &rados; objects available on
+      that pool:
+     </para>
+<screen>&prompt.cephuser;rados -p cephfs_data -N ganesha ls
+conf-osd-node1
+export-1
+conf-osd-node2</screen>
+     <tip>
+      <para>
+       To see how each node is configured, just view its content:
+      </para>
+<screen>&prompt.cephuser;cat conf-osd-node1
+%url "rados://cephfs_data/ganesha/export-1"
+cat conf-osd-node2
+%url "rados://cephfs_data/ganesha/export-1"</screen>
+      <para>
+       In this case, both nodes will use
+       <literal>rados://cephfs_data/ganesha/export-1</literal>. If there are
+       multiple configurations, each node can use a different configuration.
+      </para>
+     </tip>
+    </step>
+    <step>
+     <para>
+      Each export configuration is stored in a single object with the name
+      <literal>export-<replaceable>ID</replaceable></literal>. Use the
+      following command to get the contents of the object and save it to
```

Comment:
Same comment about "to get" applies here.

Reply:
i know, maybe 'obtain' is generally better here?

Reply:
Yep, I like that!

Reply:
Agreed. "Get" is a very vague word which has dozens of meanings. How about "retrieve" or "obtain"?